tests/sanity/ignore-2.10.txt

```diff
@@ -0,0 +1 @@
+plugins/modules/route53.py validate-modules:parameter-state-invalid-choice # route53_info needs improvements before we can deprecate this
```

Comment:
Looks like this change also needs to be in the other ignore files.

Reply:
Done!